app/javascript/psulib_blacklight/javascripts/libraries_locations.json

```
@@ -1,716 +1,716 @@
+    "PENNA-DP": "Penn State Law (UP) - 2nd Fl. - Penna. Collection",
+    "PENN-EM": "Earth & Min. Sci., 105 Deike Bldg - Pennsylvania Reference",
+    "PENNSYLVAN": "Wilkes-Barre - Pennsylvania Collection",
+    "PERBND-DN": "Dickinson Law (Carlisle) - 2nd Floor - Periodicals",
+    "PERBND-DP": "Penn State Law (UP) - 3rd Floor - Periodicals",
+    "PERIOD-AA": "Altoona - Periodicals",
+    "PERIOD-AH": "Arts & Humanities, Pattee, 2nd Floor - Current Periodicals",
+    "PERIODARCV": "Univ. Archives, 1st Floor Paterno, Current Periodicals",
+    "PERIOD-BD": "Erie - Periodicals",
+    "PERIOD-BU": "Business, 3rd Paterno - Current Periodicals - In-Library Use",
+    "PERIOD-DN": "Dickinson Law (Carlisle) - 2nd Floor - Periodicals",
+    "PERIOD-DP": "Penn State Law (UP) - 3rd Floor - Periodicals",
+    "PERIOD-ED": "Paterno, 5th Floor - Current Periodicals Area",
+    "PERIOD-GV": "Great Valley - Periodicals - In-Library Use",
+    "PERIOD-HB": "Harrisburg - Periodicals - 2nd Floor - In-Library Use",
+    "PERIOD-LS": "Paterno, 4th Floor - Current Periodicals Area",
+    "PERIOD-NML": "News & Microforms, Leisure Reading Room - Magazine Area",
+    "PERIOD-PT": "Paterno - 1st Floor - Magazine Display Area",
+    "PERIOD-RH": "Paterno, 3rd Floor - Current Periodicals Area",
+    "PERIOD-SS": "Soc. Sci., 2nd Paterno - Current Periodicals - In-Library Use",
+    "PEROVER-DN": "Dickinson Law (Carlisle) - 2nd Floor - Oversize Periodicals",
+    "PEROVER-DP": "Penn State Law (UP) - 3rd Floor -Oversized Periodicals",
+    "PHONO-HB": "Harrisburg - Phonograph Record Collection - 1st Floor",
+    "PLANTPATEN": "Life Sciences, 4th Paterno - Plant Patents - In-Library Use",
+    "POPNON-NK": "New Kensington - Popular Non-Fiction",
+    "POPSCI-PM": "Phys. & Math. Sci., 201 Davey Lab - Popular Science",
+    "POPULAR-EG": "Engineering, 325 Hammond Bldg. - Popular Reading in Tech.",
+    "PRESLAB-UP": "Preservation Department, Conservation Lab - Reference Coll.",
+    "PRESREF-UP": "Preservation Department, 402 Pattee Library - Reference Coll",
+    "PRIESTLEY": "Rare Books & Mss, 1st Floor Paterno, Priestley Collection",
+    "PROFDEV-EG": "Engineering, 325 Hammond Bldg. - Professional Development",
+    "PSUARC-GCS": "Obsolete Location - Do Not Use",
+    "PSUARCHIVE": "Univ. Archives, 1st Floor Paterno",
+    "PSUARC-MSS": "Univ. Archives, 1st Floor Paterno, Univ. Archives & Mss",
+    "PSUPRESS": "Univ. Archives, 1st Floor Paterno, Univ. Press Coll.",
+    "PUGH": "Univ. Archives, 1st Floor Paterno, Pugh Collection",
+    "PURPLEQUIL": "Univ. Archives, 1st Floor Paterno, Purple Quill Coll.",
+    "RAMP-DP": "Penn State Law (UP) - Carrel Ramp",
+    "RAREBKS-HB": "Harrisburg - Rare Books - Special Coll - 3rd Floor",
+    "RARE-BOOK": "Rare Books & Mss, 1st Floor Paterno",
+    "RARE-CHILD": "Rare Books & Mss, 1st Floor Paterno, Children's Book Coll.",
+    "RARE-DN": "Dickinson Law (Carlisle) - Rare Book R
```

```suggestion
    "RESERVE-AC": "Architecture & Landscape Arch, Stuckeman Bldg – Open Reserve",
```
@ruthtillman replacing this one with a `–` as well